File: app/src/main/java/com/soft/credit911/ui/Chat/Fragment/Adapter/MessageViewHolder.java
package com.soft.credit911.ui.Chat.Fragment.Adapter;

import android.view.View;

import androidx.recyclerview.widget.RecyclerView;
import com.soft.credit911.ui.Chat.Fragment.Model.ChatObject;

public abstract class MessageViewHolder extends RecyclerView.ViewHolder {

    MessageViewHolder(View view) {
        super(view);
    }
    public abstract void onBindView(ChatObject object);
}

File: app/src/main/java/com/soft/credit911/adaptor/MessageViewHolder.kt
package com.soft.credit911.adaptor

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import com.soft.credit911.datamodel.ChatObject

abstract class MessageViewHolder internal constructor(view: View?) : RecyclerView.ViewHolder(
    view!!
) {
    abstract fun onBindView(`object`: ChatObject?)
}